qa/integration/fixtures/logstash_to_logstash_spec.yml

```diff
@@ -22,6 +22,15 @@ config:
       generator {
         count => '<%=options[:generator_count]%>'
       }
+      heartbeat {
```

Comment:
Please add a comment that specify the reason why it's needed

Reply:
This PR replaces this with reload: https://github.com/elastic/logstash/pull/17246